bigquery/google/cloud/bigquery/dataset.py

```diff
@@ -269,8 +278,9 @@ def from_string(cls, dataset_id, default_project=None):
         Args:
             dataset_id (str):
                 A dataset ID in standard SQL format. If ``default_project``
-                is not specified, this must included both the project ID and
-                the dataset ID, separated by ``.``.
+                is not specified, this must include both the project ID and
+                the dataset ID, separated by ``.`` or, single prefix usage is
```

Comment:
I find the "single prefix" term to be confusing. I can't find any documentation on these kinds of project IDs. I suggest reverting the changes to the docstrings, since it will confuse most customers.